app/app.js

```diff
@@ -103,7 +103,10 @@ if (module.hot) {
 
 // Chunked polyfill for browsers without Intl support
 if (!window.Intl) {
-  System.import('intl').then(() => render(translationMessages));
+  Promise.all([
+    System.import('intl'),
+    System.import('intl/locale-data/jsonp/en.js'),
```

Comment:
Why do we import the `en` local-data here? Do we need to do the same thing for all languages?